bitcoin/src/crypto/sighash.rs

```diff
@@ -515,6 +529,25 @@ impl TapSighashType {
             x => return Err(InvalidSighashTypeError(x as u32)), // Cast ok, u8 -> u32
         })
     }
+
+    /// Converts [`EcdsaSighashType`] to a `u8` sighash flag.
+    ///
+    /// The returned value is guaranteed to be a valid according to standardness rules.
+    pub fn to_u8(self) -> u8 { self as u8 }
+
+    /// Converts [`EcdsaSighashType`] to a `u32` sighash flag.
+    ///
+    /// The returned value is guaranteed to be a valid according to standardness rules.
+    pub fn to_u32(self) -> u32 { self as u32 }
+}
+
+impl From<TapSighashType> for u8 {
+    // Cast ok because sighash types are only one byte.
+    fn from(s: TapSighashType) -> u8 { s.to_u8() }
+}
+
+impl From<TapSighashType> for u32 {
```

Comment:
No `u32`

Reply:
This is outdated but I'm just flagging that I don't understand this. I don't know if this means "no don't do it for `u32`" or "missing `From<u32> for TapSighashType`.